src/applications/timeouts.ts

```diff
@@ -150,7 +184,7 @@ export function reasonableTime(appOrParcel, lifecycle) {
             //don't resolve or reject, we're waiting this one out
           }
         } else if (!errored) {
-          const numWarnings = shouldError;
+          const numWarnings: number = shouldError as number;
```

Comment:
The `as number` type cast here should be safe, due to the if statement on line 178

Reply:
Is this cast necessary? For me, typescript seems to correctly infer that `shouldError` is a `number` here.

Reply:
Good catch, I've removed it